FacLib/Employee.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Reflection.Metadata.Ecma335;
+
+namespace EmpLib
+{
+    public class Employee
+    {
+       
+        private string firstName;
```

Comment:
private variable naming is incorrect
Fix according to [C# Coding Style](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md)

Reply:
fix everywhere